src/WORKSPACE

```diff
@@ -61,6 +61,14 @@ new_git_repository(
     remote = "https://github.com/saebyn/munkres-cpp.git",
 )
 
+# yaml cpp parser for dynamic parameters test
+git_repository(
+    name = "yaml-cpp",
+    commit = "f531f8a8c3f8ca0923f9219c72fcca511555f824",
+    remote = "https://github.com/jbeder/yaml-cpp.git",
+    shallow_since = "1571445859 -0700",
```

Comment:
can we clarify why this commit?

Reply:
I just picked the most recent commit to freeze it there, these is no specific reason other than that I know the test passes w/ this library

Reply:
let's add a comment to make that clear

Reply:
I went to write `# commit points to last known working version of library` but I don't know if that adds too much value as that is the intention of specifying the commit hash. If you think otherwise I can go back and add it as it doesn't do any harm

Reply:
Well what if someone wants to change it in the future for some reason they would want to know that this is an arbitrary commit and we're not using this commit for a specific reason

Reply:
That's what we're doing for all the external stuff here, should we add comments for all of them? Seems a bit overkill, it's fairly standard bazel convention to always fix your commits so the build remains hermetic.

Reply:
Ok so the default assumption is that it's some arbitrary commit